### PR TITLE
chore: adapt teamsfx package

### PR DESCRIPTION
**Pull request by IvanJobs**
cli's sub-command is changed, adapt it inside ymls.

to finish https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_sprints/taskboard/Compute%20Team/Microsoft%20Teams%20Extensibility/CY21-8.2?workitem=10629315

### CODE REVIEW

File: docs/cicd/github-cd-template.yml [hunk 1]
```diff
@@ -80,10 +80,10 @@ jobs:
 
       # This step is to pack the Teams App as zip file,
       # which can be used to be uploaded onto Teams Client for installation.
-      - name: Build Teams App's package
+      - name: Package your Teams App for publishing
```
Comment (henzhang-ms) [inline docs/cicd/github-cd-template.yml:83]: Nit: a little bit weird to see `your` in the action name.

**Review by henzhang-ms — APPROVED** [review]
Approve

File: docs/cicd/github-cd-template.yml (hunk 1 above)
Comment (MuyangAmigo) [reply docs/cicd/github-cd-template.yml:83]: Agree, `Package Teams App for publishing` might be better.

Comment (IvanJobs) [reply docs/cicd/github-cd-template.yml:83]: @Alive-Fish For awareness, borrow it from cli's help doc. I will update the yml soon.

**Pull request merged**